app/src/main/res/drawable/rounded_button_background.xml

```diff
@@ -12,7 +12,7 @@
     </item>
     <item>
         <shape android:shape="rectangle">
-            <solid android:color="?inset" />
+            <solid android:color="?actionPrimary" />
```

Comment:
`rounded_button_background.xml` is only used for Primary/Positive Buttons. See https://searchfox.org/mozilla-mobile/search?q=%40drawable%2Frounded_button_background&path= for usage. So, I intentionally just set it to the Action Primary color. Refer to https://www.figma.com/file/YcQC7rU1SQJx49kIAXZhcI/Component-and-Color-Usage-Examples?node-id=78%3A19170 for an example of what Action Primary buttons are.